### PR TITLE
Corrected grammatical errors, improved sentence structures, translation for migration 2.x-3.x

### PR DESCRIPTION
**Pull request by Latand**
Corrected grammatical errors, improved sentence structures, translation for migration 2.x-3.x

# Description

Corrected grammatical errors, improved sentence structures for clarity across various sections of the aiogram 3 library documentation.

## Type of change

Please delete options that are not relevant.

- [x] Documentation (typos, code examples or any documentation update)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/migration_2_to_3.rst [hunk 1]
```diff
@@ -102,41 +109,43 @@ Keyboard Markup
...
+  instead of passing the IO object directly to the API method. (:ref:`Read more » <sending-files>`)
 
 
 Webhook
 =======
 
-- Simplified aiohttp web app configuration
-- By default added possibility to upload files when you use reply into webhook
+- The aiohttp web app configuration has been simplified.
+- By default, the ability to upload files has been added when you use the reply function in a webhook.
```
Comment (Olegt0rr) [inline docs/migration_2_to_3.rst:144]:
```suggestion
- By default, the ability to upload files has been added when you `make requests in response to updates <https://core.telegram.org/bots/faq#how-can-i-make-requests-in-response-to-updates>`_ (available for webhook only).
```

File: docs/migration_2_to_3.rst [hunk 2]
```diff
@@ -24,76 +25,82 @@ On this page you can read about points that changed corresponding to last stable
...
-- Most of common filters is replaced by "magic filter". (:ref:`Read more » <magic-filters>`)
-- Now by default message handler receives any content type,
-  if you want specific one just add the filters (Magic or any other)
-- State filter now is not enabled by default, that's mean if you using :code:`state="*"` in v2
-  then you should not pass any state filter in v3, and vice versa,
-  if the state in v2 is not specified now you should specify the state.
-- Added possibility to register per-router global filters, that helps to reduces
-  the number of repetitions in the code and makes easily way to control
-  for what each router will be used.
+- Most common filters have been replaced by the "magic filter." (:ref:`Read more » <magic-filters>`)
```
Comment (Olegt0rr) [inline docs/migration_2_to_3.rst:57]:
```suggestion
- Most common filters have been replaced with the "magic filter." (:ref:`Read more » <magic-filters>`)
```

Comment (Latand) [reply docs/migration_2_to_3.rst:57]: Oh yeah, correct

File: docs/locale/uk_UA/LC_MESSAGES/api/methods/set_sticker_set_thumb.po [hunk 3]
```diff
@@ -3,48 +3,70 @@
...
...**.TGS** з мініатюрою розміром до 32 кілобайт (див. `https://core.telegram.org/stickers#animated-sticker-requirements <https://core.telegram.org/stickers#animated-sticker-requirements>`_`https://core.telegram.org/stickers#animated-sticker-requirements <https://core.telegram. org/stickers#animated-sticker-requirements>`_ технічні вимоги до анімованих наліпок), або **WEBM** відео з мініатюрою розміром до 32 кілобайт; дивіться `https://core.telegram.org/stickers#video-sticker-requirements <https://core.telegram.org/stickers#video-sticker-requirements>`_`https://core.telegram.org/stickers#video-sticker-requirements <https://core.telegram.org/stickers#video-sticker-requirements>`_ технічні вимоги до відео наліпок. Передайте *file_id* як рядок, щоб надіслати файл, який вже існує на серверах Telegram, передайте HTTP URL як рядок, щоб Telegram отримав файл з Інтернету, або завантажте новий файл за допомогою мультичастини/форма-даних. :ref:`Додаткова інформація про надсилання файлів \" <sending-files>`. Ескізи анімованих і відео-наборів наклейок не можуть бути завантажені через HTTP URL. Якщо цей параметр не вказано, то мініатюру буде вилучено, а замість неї буде використано першу наліпку."
```
Comment (MelissaLemon) [inline docs/locale/uk_UA/LC_MESSAGES/api/methods/set_sticker_set_thumb.po:50]: Зображення у форматі **.WEBP** або **.PNG** з мініатюрою має бути розміром до 128 кілобайт і мати ширину та висоту рівно 100 пікселів, або АНІМАЦІ**ЄЮ** у форматі...

File: docs/migration_2_to_3.rst [hunk 4]
```diff
@@ -24,76 +25,82 @@ On this page you can read about points that changed corresponding to last stable
...
   thing named :class:`Router` (:ref:`Read more » <Nested routers>`).
-  With routes you can easily separate your code to multiple modules
-  and may be share this modules between projects.
+- With routes, you can easily modularize your code and potentially share these modules between projects.
 - Removed the **_handler** suffix from all event handler decorators and registering methods.
   (:ref:`Read more » <Event observers>`)
-- Executor entirely removed, now you can use Dispatcher directly to start polling or webhook.
-- Throttling method is completely removed, now you can use middlewares to control
-  the execution context and use any throttling mechanism you want.
+- The Executor has been entirely removed; you can now use the Dispatcher directly to start polling or handle webhooks.
```
Comment (Olegt0rr) [inline docs/migration_2_to_3.rst:37]: Can we convert the phrases to a single form?

```suggestion
- The Executor has been entirely removed; you can now use the Dispatcher directly to start polling or webhook handling.
```

or

```suggestion
- The Executor has been entirely removed; you can now use the Dispatcher directly to start poll updates or handle webhooks.
```

?

Comment (Latand) [reply docs/migration_2_to_3.rst:37]: This one is better👍

File: docs/locale/uk_UA/LC_MESSAGES/api/methods/set_sticker_set_thumb.po (hunk 3 above)
Comment (Olegt0rr) [reply docs/locale/uk_UA/LC_MESSAGES/api/methods/set_sticker_set_thumb.po:50]:
```suggestion
msgstr "Зображення у форматі **.WEBP** або **.PNG** з мініатюрою має бути розміром до 128 кілобайт і мати ширину та висоту рівно 100 пікселів, або анімацєю у форматі **.TGS** з мініатюрою розміром до 32 кілобайт (див. `https://core.telegram.org/stickers#animated-sticker-requirements <https://core.telegram.org/stickers#animated-sticker-requirements>`_`https://core.telegram.org/stickers#animated-sticker-requirements <https://core.telegram. org/stickers#animated-sticker-requirements>`_ технічні вимоги до анімованих наліпок), або **WEBM** відео з мініатюрою розміром до 32 кілобайт; дивіться `https://core.telegram.org/stickers#video-sticker-requirements <https://core.telegram.org/stickers#video-sticker-requirements>`_`https://core.telegram.org/stickers#video-sticker-requirements <https://core.telegram.org/stickers#video-sticker-requirements>`_ технічні вимоги до відео наліпок. Передайте *file_id* як рядок, щоб надіслати файл, який вже існує на серверах Telegram, передайте HTTP URL як рядок, щоб Telegram отримав файл з Інтернету, або завантажте новий файл за допомогою мультичастини/форма-даних. :ref:`Додаткова інформація про надсилання файлів \" <sending-files>`. Ескізи анімованих і відео-наборів наклейок не можуть бути завантажені через HTTP URL. Якщо цей параметр не вказано, то мініатюру буде вилучено, а замість неї буде використано першу наліпку."
```

File: docs/dispatcher/errors.rst [hunk 5]
```diff
@@ -15,13 +15,13 @@ If you specify errors handler for dispatcher - it will be used for all handlers
 
 .. code-block:: python
 
-    @router.error(ExceptionTypeFilter(MyCustomException), F.update.message.as_("message"))
+    @router.errors(ExceptionTypeFilter(MyCustomException), F.update.message.as_("message"))
```
Comment (JrooTJunior) [inline docs/dispatcher/errors.rst:18]: error**s** is planned to be reomoved (use error instead), all other events are indicated in the singular

**Review by JrooTJunior — CHANGES_REQUESTED** [review]

File: docs/dispatcher/errors.rst (hunk 5 above)
Comment (Latand) [reply docs/dispatcher/errors.rst:18]: Wow... Isn't it a breaking change?

Comment (Latand) [reply docs/dispatcher/errors.rst:18]: done

**Review by JrooTJunior — APPROVED** [review]

**Pull request merged**